ui/css/itcss/components/send.scss

```diff
@@ -1,927 +0,0 @@
-@use "design-system";
-
-.send-screen-wrapper {
-  display: flex;
-  flex-flow: column nowrap;
-  z-index: 25;
-
-  @include design-system.screen-sm-max {
-    width: 100%;
-    overflow-y: auto;
-  }
-
-  section {
-    flex: 0 0 auto;
-  }
-}
-
-.send-screen-card {
-  background-color: var(--color-background-default);
-  box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
-  padding: 46px 40.5px 26px;
-  position: relative;
-  align-items: center;
-  display: flex;
-  flex-flow: column nowrap;
-  width: 498px;
-  flex: 1 0 auto;
-
-  @include design-system.screen-sm-max {
-    top: 0;
-    width: 100%;
-    box-shadow: none;
-    padding: 12px;
-  }
-}
-
-/* Send Screen */
-
-.send-screen section {
-  margin: 4px 16px;
-}
-
-.send-screen input {
-  @include design-system.H7;
-
-  width: 100%;
-}
```

Comment:
Not used

![Screenshot 2025-02-07 at 11 56 59 AM](https://github.com/user-attachments/assets/6c8a8f79-7f04-45bc-a468-822826d5dbae)